cmd/podman/containers/run.go

```diff
@@ -76,8 +76,12 @@ func runFlags(cmd *cobra.Command) {
 	detachKeysFlagName := "detach-keys"
 	flags.StringVar(&runOpts.DetachKeys, detachKeysFlagName, containerConfig.DetachKeys(), "Override the key sequence for detaching a container. Format is a single character `[a-Z]` or a comma separated sequence of `ctrl-<value>`, where `<value>` is one of: `a-cf`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`")
 	_ = cmd.RegisterFlagCompletionFunc(detachKeysFlagName, common.AutocompleteDetachKeys)
+
+	flags.StringVar(&runOpts.PidFile, "pidfile", "", "Write the container process ID to the file")
```

Comment:
You should define this flag in  `./cmd/podman/common/create.go`. This file contains flag for podman create and run so you don't have to add them twice.

Reply:
Move the flag to  `./cmd/podman/common/create.go`